pagopa/io-app
# feat(Firma con IO): [SFEQS-2083] Replace abort bottom sheet with alert

## Short description
This PR replaces the abort bottom sheet with an alert when using the new design system, according to this [FIGMA](https://www.figma.com/file/bX1jqH6ZHpAwoZgiF9M7Io/Firma-con-IO---2.0?type=design&node-id=68-9156&mode=design&t=QWsS3Pf7OJALcyHh-0). 
The shown alert is just a placeholder for a nativa alert, thus we are leaving Apple and OEMs for Android to decide how it looks on each device, including the order of the buttons.
On iOS the `style: 'cancel'` prop places the button as second when text requires wrapping. On Android, two buttons means `negative`, `positive` and are placed accordingly ([ref](https://reactnative.dev/docs/alert)). 

## List of changes proposed in this pull request
- Add locales;
- _Override_ the `present` function of the bottom sheet to show an alert instead if the experimental design is enabled.
This allows us to use an alert without changing single components which use the hook.
The bottom sheet will be removed once the experimental design is enabled by default ([SFEQS-2090]([SFEQS-2090](https://pagopa.atlassian.net/browse/SFEQS-2090))).

## How to test
Open a signature flow either in production or with `io-dev-api-server` by setting the flag `waitForSignatureCount: 1`. Then try to abort it. Either a bottom sheet or an alert should be displayed if the experimental design system is disabled or not.



[SFEQS-2090]: https://pagopa.atlassian.net/browse/SFEQS-2090?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SFEQS-2090]: https://pagopa.atlassian.net/browse/SFEQS-2090?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
